File: Sprint_01/Atividade 1/1.5/tarefa.py
pcGasolina = -1
pcAlcool = -1

while(pcGasolina <= 0):
	pcGasolina = float(input('Informe o preço da Gasolina: '))

while(pcAlcool <= 0):
	pcAlcool = float(input('Informe o preço do Alcool: '))

calculoDiferenca =  pcAlcool / pcGasolina

if calculoDiferenca > 0.70:
	print('A gasolina é melhor!', calculoDiferenca)
else:
	print('O alcool é melhor!', calculoDiferenca)

File: Sprint 01/Atividade 1/1.5/Tarefa.java
import java.util.Scanner;

public class Tarefa {
	public static void main(String[] args) {
		double pcGasolina, pcAlcool, calculoDiferenca;
		Scanner scanner = new Scanner(System.in);

		do{
			System.out.print("Informe o preco da Gasolina: ");
			pcGasolina = Double.parseDouble(scanner.next());
		}while(pcGasolina <= 0);

		do{
			System.out.print("Informe o preco do Alcool: ");
			pcAlcool = Double.parseDouble(scanner.next());
		}while(pcAlcool <= 0);

		calculoDiferenca = pcAlcool / pcGasolina;

		if(calculoDiferenca > 0.70)
			System.out.println(calculoDiferenca + " - Gasolina eh melhor!");
		else
			System.out.println(calculoDiferenca + " - Alcool eh melhor!");

		scanner.close();
	}
}
